src/main/java/ch/njol/skript/expressions/ExprRawString.java

```diff
@@ -71,14 +72,21 @@ public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelaye
 	}
 
 	@Override
-	protected String[] get(Event e) {
+	protected String[] get(Event event) {
 		List<String> strings = new ArrayList<>();
 		for (Expression<? extends String> message : messages) {
 			if (message instanceof VariableString) {
-				strings.add(((VariableString) message).toUnformattedString(e));
+				strings.add(((VariableString) message).toUnformattedString(event));
 				continue;
 			}
-			strings.addAll(Arrays.asList(message.getArray(e)));
+			for (String string : message.getArray(event)) {
+				String raw = SkriptColor.replaceColorChar(string);
+				if (raw.contains("&x")) {
```

Comment:
Even tho `&x` is hardcoded within Skript but for comparison it should be made on lowercased `raw`